draft-ietf-quic-multipath.md

```diff
@@ -112,11 +112,8 @@ non-zero connection IDs in order to identify the path and respective
 packet number space.
 
 To add a new path to an existing QUIC connection with multipath support,
-a client starts a path validation on
+a client starts a path validation or sends a packet on
 the chosen path, as further described in {{path-initiation}}.
-A new path can only be used once the associated 4-tuple has been validated
-by ensuring that the peer is able to receive packets at that address
-(see {{Section 8 of QUIC-TRANSPORT}}).
```

Comment:
Why did you remove this sentence? I think t's correct and simply part of the summary overview at the beginning. It is still true that a path/address as to be validated before using it. It's just the case that we a) can't enforce that validation by the other end and b) there can be other way to validate than sending a path challenge. However, this sentence doesn't say anything about these two points.

Reply:
I will try to rewrite.

Reply:
Rewritten in
[Rewrite per review feedback.](https://github.com/quicwg/multipath/pull/442/commits/9f74f86bb43f5509f4f49e60e61aac40a1f6679a)